topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to recurrent neural networks (RNN)
+zenodo_link: https://zenodo.org/record/4477881
+questions:
+- What is a recurrent neural network (RNN)?
+- What are some applications of RNN?
+objectives:
+- Understand the difference between feedforward neural networks (FNN) and RNN
+- Learn various RNN types and architectures
+- Solve a sentiment analysis problem on IMDB movie review dataset using RNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Artificial neural networks are a machine learning discipline roughly inspired by how neurons in a 
+human brain work. In the past decade, there has been a huge resurgence of neural networks thanks 
+to the vast availability of training data and enormous increases in computing capacity (Successfully 
```

Comment:
`data` should be sufficient instead of `training data` because the latter leads towards the concept of `training/test data`

Reply:
Sure,